tests/pyright_test.py

```diff
@@ -16,26 +14,12 @@ def main() -> None:
         print("pyright_test.py must be run from the typeshed root directory", file=sys.stderr)
         sys.exit(1)
 
-    # subprocess.run on Windows does not look in PATH.
-    npx = shutil.which("npx")
-
-    if npx is None:
-        print("error finding npx; is Node.js installed?", file=sys.stderr)
-        sys.exit(1)
-
-    try:
-        subprocess.run([npx, "--version"])
-    except OSError:
-        print("error running npx; is Node.js installed?", file=sys.stderr)
-        sys.exit(1)
-
-    with open("pyproject.toml", "rb") as config:
-        pyright_version: str = tomli.load(config)["tool"]["typeshed"]["pyright_version"]
-
-    os.environ["PYRIGHT_PYTHON_FORCE_VERSION"] = pyright_version
-    command = [npx, f"pyright@{pyright_version}"] + sys.argv[1:]
-    print("Running:", " ".join(command))
+    print(colored("Running: pyright --version", "blue"))
```

Comment:
Why blue? I don't think we print anything else in blue/info/note color specifically (and if it's something we wanna do, could be done separately)